pages/give-devices.js

```diff
@@ -9,21 +9,19 @@ import SelectFormControl from '../components/form/SelectFormControl';
 import TextareaFormControl from '../components/form/TextareaFormControl';
 
 // Use ISO 3166 country codes per https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
-const COUNTRIES = [
-  { value: 'US', label: 'United States' },
-  { value: 'CA', label: 'Canada' },
-  { value: 'MX', label: 'Mexico' },
-  { value: 'DE', label: 'Germany' },
-  { value: 'LV', label: 'Latvia' },
-  { value: 'IN', label: 'India' },
-  { value: 'BD', label: 'Bangladesh' },
-  { value: 'PH', label: 'Philippines' },
-];
+const countryList = require('country-list');
+
+const countries = countryList.getData();
+countries.sort((a, b) => (a.name > b.name && 1) || -1);
```

Comment:
Just a thought this line sorts the values every time we visit this page, can it cause any performance issue?
we can create a JSON file with sorted order then use it every time we want to load the data, just an idea.

Reply:
Apology for the unnecessary {' '}, this was due to some issue with my code formatter I have removed them.

Reply:
No worries great job on this PR!!!! ;)